indico/modules/events/registration/controllers/api/misc.py

```diff
@@ -5,10 +5,19 @@
 # modify it under the terms of the MIT License; see the
 # LICENSE file for more details.
 
+from itertools import groupby
+
 from indico.modules.events.controllers.base import RHProtectedEventBase
 
 
 class RHAPIRegistrationForms(RHProtectedEventBase):
     def _process(self):
         from indico.modules.events.registration.schemas import RegistrationFormPrincipalSchema
         return RegistrationFormPrincipalSchema(many=True).jsonify(self.event.registration_forms)
+
+
+class RHAPIEventTimeTable(RHProtectedEventBase):
+    def _process(self):
+        schema = [{'id': sb.id, 'start_dt': sb.start_dt.strftime('%d-%b-%Y'),
```

Comment:
That's not a date format which belongs in an API.
APIs should always return dates using `.isoformat` (yyyy-mm-dd); you can then convert it to something different on the client side.

Also, I think you need to use `sb.start_dt.astimezone(self.event.tzinfo)` to get the correct date depending on the event's timezone.

Reply:
I agree, but  I still need the date, so i guess it should be `sb.start_dt.astimezone(self.event.tzinfo).date().isoformat()` am I right?